src/shared/components/msktable/MSKTable.tsx

```diff
@@ -102,10 +123,46 @@ class MSKTableStore<T> {
         return tableDownloadData;
     }
 
+    @computed get sortColumnObject():Column<T>|undefined {
+        return this.columns.find((col:Column<T>)=>this.isVisible(col) && (col.name === this.sortColumn));
+    }
+
+    @computed get sortedData():T[] {
+        let ret = this.data;
+        const column = this.sortColumnObject;
+        if (column) {
+            if (column.sortValue) {
+                switch (column.sortType) {
```

Comment:
the reason for this was I was multiplexing to which default sort method to use. but this problem may be obsolete bc _.sortBy handles every type of argument